lib/tapioca/commands/abstract_dsl.rb

```diff
@@ -131,15 +131,22 @@ def load_application
 
       sig { returns(Tapioca::Dsl::Pipeline) }
       def create_pipeline
+        error_handler = if @lsp_addon
+          ->(error) {
+            say(error, :bold, :red)
```

Comment:
This is an adjacent change to show errors in lsp output. `say_error` messages aren't visible in the logs, I assume due to `stderr`, but we'd like to have them.

Reply:
Good point.